src/components/content/content.test.js

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import Content from "./content.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import {
+  contentTitle,
+  contentBody,
+  contentPreview,
+} from "../../../cypress/locators/content/index";
+
+context("Tests for Content component", () => {
+  describe("should check Content component properties", () => {
+    it.each(["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"])(
+      "should check %s as children for Content component",
+      (children) => {
+        CypressMountWithProviders(<Content title="Title">{children}</Content>);
+        contentBody().should("have.text", children);
+      }
+    );
+
+    it.each([
+      ["mp150ú¿¡üßä", "primary"],
+      ["!@#$%^*()_+-=~[];:.,?{}&\"'<>", "secondary"],
+    ])("should check %s as title for Content component", (title, variant) => {
+      CypressMountWithProviders(
+        <Content title={title} variant={variant}>
```

Comment:
we need to check only one param in each test. For example you are asserting here a `title` so there is no need to put here `variant` if you are not checking it later